llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py

```diff
@@ -138,7 +139,8 @@ def from_documents(  # type: ignore
             else:
                 if verbose:
                     print(".", end="")
-                time.sleep(0.5)
+                # Sleep for 0 seconds to release the GIL and allow other threads to run
```

Comment:
Oh, this sleep (and the one after it) is there so that we don't ping the api for results a million times lol

Reply:
Removed.